src/SosThreadingTools/Commands.cs

```diff
@@ -38,7 +38,15 @@ private static void ExecuteCommand(IntPtr client, string command, [MarshalAs(Unm
                 return;
             }
 
-            handler.Execute(context, args);
+            try
+            {
+                handler.Execute(context, args);
+            }
+            catch (Exception ex)
+            {
+                context.Output.WriteLine($"Encountered an unhandled exception running '{command}':");
+                context.Output.WriteLine(ex.ToString());
```

Comment:
Should we not still re-throw the exception rather than swallow it?

Reply:
It shouldn't matter.  WinDbg has a giant try catch which will swallow the exception, printing out an unhelpful error message.  Personally, I tend to try to not throw C# exceptions that are caught in C++ code (even though it works fine).

If you prefer it the other way, I'd be happy to change this to add "throw;".  Just let me know your preference.

Reply:
Ok, fair enough.